ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -329,3 +329,36 @@ def test_numpy_diff(
         x=x[0],
         axis=axis,
     )
+    
+    
+# ediff1d
+@handle_frontend_test(
+    fn_tree="numpy.ediff1d",
+    dtype_x_axis=helpers.dtype_values_axis(
```

Comment:
You are not using axis anywhere, so I would suggest using `helpers.dtype_and_values`.